core/src/test/java/com/amazon/opendistroforelasticsearch/sql/expression/operator/predicate/BinaryPredicateOperatorTest.java

```diff
@@ -458,46 +458,46 @@ public void test_less_null() {
     FunctionExpression less = dsl.less(DSL.literal(1),
         DSL.ref(INT_TYPE_NULL_VALUE_FIELD, INTEGER));
     assertEquals(BOOLEAN, less.type());
-    assertEquals(LITERAL_NULL, less.valueOf(valueEnv()));
+    assertEquals(LITERAL_FALSE, less.valueOf(valueEnv()));
```

Comment:
So `1 < NULL` returns `FALSE` now?

Reply:
This is the default order in our system.
NULL equal to NULL
NULL large than MISSING
NULL less than all others.

In Order By, the user can also specify the order of NULL and MISSING.

Reply:
Got your point, change the logic, follow the MySQL way now.
https://dev.mysql.com/doc/refman/8.0/en/working-with-null.html